### PR TITLE
add `txn-entry-size-limit` some desc

### PR DESCRIPTION
**Pull request by glkappe**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by iosmanthus** [comment]
Seem the CI reported some markdown linting issue. Please check it out.

File: tidb-configuration-file.md [hunk 1]
```diff
@@ -339,7 +339,10 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
 + 默认值：6291456
 + 单位：Byte
 + 事务中单个 key-value 记录的大小限制。若超出该限制，TiDB 将会返回 `entry too large` 错误。该配置项的最大值不超过 `125829120`（表示 120MB）。
-+ 注意，TiKV 有类似的限制。若单个写入请求的数据量大小超出 [`raft-entry-max-size`](/tikv-configuration-file.md#raft-entry-max-size)，默认为 8MB，TiKV 会拒绝处理该请求。当表的一行记录较大时，需要同时修改这两个配置。
+> **注意：**
```
Comment (Oreoxmt) [inline tidb-configuration-file.md:342]:
```suggestion

> **注意：**
```

**Comment by TomShawn** [comment]
@iosmanthus PTAL again, thanks

**Comment by qiancai** [comment]
Removed the needs-cherry-pick-release-6.6 label because the v6.6 docs have been archived at https://docs-archive.pingcap.com/zh/tidb/v6.6 and will no longer receive new updates.

**Comment by qiancai** [comment]
Removed the needs-cherry-pick-release-7.0 label because the v7.0 docs have been archived at https://docs-archive.pingcap.com/zh/tidb/v7.0 and will no longer receive new updates.

**Comment by qiancai** [comment]
Removed the `needs-cherry-pick-release-7.2` label because the v7.2 docs have been archived at https://docs-archive.pingcap.com/zh/tidb/v7.2 and will no longer receive new updates.

**Review by overvenus — APPROVED** [review]

File: tidb-configuration-file.md [hunk 2]
```diff
@@ -495,12 +495,18 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
...
-+ 事务中单个 key-value 记录的大小限制。若超出该限制，TiDB 将会返回 `entry too large` 错误。该配置项的最大值不超过 `125829120`（表示 120MB）。
++ 事务中单个 key-value 记录的大小限制。若超出该限制，TiDB 将会返回 `entry too large` 错误。该配置项的最大值不超过 `125829120`（表示 120 MiB）。
 + 从 v7.6.0 开始，你可以使用 [`tidb_txn_entry_size_limit`](/system-variables.md#tidb_txn_entry_size_limit-从-v760-版本开始引入) 系统变量动态修改该配置项的值。
-+ 注意，TiKV 有类似的限制。若单个写入请求的数据量大小超出 [`raft-entry-max-size`](/tikv-configuration-file.md#raft-entry-max-size)，默认为 8MB，TiKV 会拒绝处理该请求。当表的一行记录较大时，需要同时修改这两个配置。
 + [`max_allowed_packet`](/system-variables.md#max_allowed_packet-从-v610-版本开始引入) (MySQL 协议的最大数据包大小) 的默认值为 `67108864`（64 MiB）。如果一行记录的大小超过 `max_allowed_packet`，该行记录会被截断。
 + [`txn-total-size-limit`](#txn-total-size-limit)（TiDB 单个事务大小限制）的默认值为 100 MiB。如果将 `txn-entry-size-limit` 的值设置为 100 MiB 以上，需要相应地调大 `txn-total-size-limit` 的值。
 
+> **注意：**
+>
+> 当表的一行记录较大时，除了修改上述配置外，还需要同时调整 TiKV 的两个配置项，以保证 TiKV 之间的通信不受影响。
```
Comment (lilin90) [inline tidb-configuration-file.md:505]:
```suggestion
> 当表的一行记录较大时，除了修改上述配置外，还需要同时调整 TiKV 的以下两个配置项，以保证 TiKV 之间的通信不受影响：
```

File: tidb-configuration-file.md [hunk 3]
```diff
@@ -495,12 +495,18 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
...
-+ 注意，TiKV 有类似的限制。若单个写入请求的数据量大小超出 [`raft-entry-max-size`](/tikv-configuration-file.md#raft-entry-max-size)，默认为 8MB，TiKV 会拒绝处理该请求。当表的一行记录较大时，需要同时修改这两个配置。
 + [`max_allowed_packet`](/system-variables.md#max_allowed_packet-从-v610-版本开始引入) (MySQL 协议的最大数据包大小) 的默认值为 `67108864`（64 MiB）。如果一行记录的大小超过 `max_allowed_packet`，该行记录会被截断。
 + [`txn-total-size-limit`](#txn-total-size-limit)（TiDB 单个事务大小限制）的默认值为 100 MiB。如果将 `txn-entry-size-limit` 的值设置为 100 MiB 以上，需要相应地调大 `txn-total-size-limit` 的值。
 
+> **注意：**
+>
+> 当表的一行记录较大时，除了修改上述配置外，还需要同时调整 TiKV 的以下两个配置项，以保证 TiKV 之间的通信不受影响：
+>
+> - [`raft-entry-max-size`](/tikv-configuration-file.md#raft-entry-max-size)，默认为 `8MiB`。若单个写入请求的数据量大小超出该限制，TiKV 会拒绝处理该请求。
+> - [`max-grpc-send-msg-len`](/tikv-configuration-file.md#max-grpc-send-msg-len)，默认为 `10485760`。
```
Comment (overvenus) [inline tidb-configuration-file.md:508]:
```suggestion
```
用户无需调整此参数，因为它不会导致写入被拒绝。

File: tidb-configuration-file.md [hunk 4]
```diff
@@ -495,12 +495,18 @@ TiDB 配置文件比命令行参数支持更多的选项。你可以在 [config/
...
-+ 事务中单个 key-value 记录的大小限制。若超出该限制，TiDB 将会返回 `entry too large` 错误。该配置项的最大值不超过 `125829120`（表示 120MB）。
++ 事务中单个 key-value 记录的大小限制。若超出该限制，TiDB 将会返回 `entry too large` 错误。该配置项的最大值不超过 `125829120`（表示 120 MiB）。
 + 从 v7.6.0 开始，你可以使用 [`tidb_txn_entry_size_limit`](/system-variables.md#tidb_txn_entry_size_limit-从-v760-版本开始引入) 系统变量动态修改该配置项的值。
-+ 注意，TiKV 有类似的限制。若单个写入请求的数据量大小超出 [`raft-entry-max-size`](/tikv-configuration-file.md#raft-entry-max-size)，默认为 8MB，TiKV 会拒绝处理该请求。当表的一行记录较大时，需要同时修改这两个配置。
 + [`max_allowed_packet`](/system-variables.md#max_allowed_packet-从-v610-版本开始引入) (MySQL 协议的最大数据包大小) 的默认值为 `67108864`（64 MiB）。如果一行记录的大小超过 `max_allowed_packet`，该行记录会被截断。
 + [`txn-total-size-limit`](#txn-total-size-limit)（TiDB 单个事务大小限制）的默认值为 100 MiB。如果将 `txn-entry-size-limit` 的值设置为 100 MiB 以上，需要相应地调大 `txn-total-size-limit` 的值。
 
+> **注意：**
+>
+> 当表的一行记录较大时，除了修改上述配置外，还需要同时调整 TiKV 的以下两个配置项，以保证 TiKV 之间的通信不受影响：
```
Comment (overvenus) [inline tidb-configuration-file.md:505]:
```suggestion
> 当表的一行记录较大时，除了修改上述配置外，还需要同时调整 TiKV 的以下配置项，以保证写入请求不会被 TiKV 拒绝：
```

**Comment by lilin90** [comment]
Since no new change exists in this PR compared with the [current description for `txn-entry-size-limit`](https://docs.pingcap.com/zh/tidb/dev/tidb-configuration-file#txn-entry-size-limit-%E4%BB%8E-v4010-%E5%92%8C-v500-%E7%89%88%E6%9C%AC%E5%BC%80%E5%A7%8B%E5%BC%95%E5%85%A5) in docs, I'll close this PR.
image

**Pull request closed without merging**